tests/unit/lib/interfaces/qunit.js

```diff
@@ -351,6 +351,18 @@ define([
 			}
 		},
 
+		'extend': {
+			'should have a working expect': function () {
+				QUnit.extend( QUnit.assert, {
```

Comment:
use a new object scoped to the test to test this utility function, there is no reason to run it against anything constructed outside of the test function.